docs/topics/jvm/java-interop.md

```diff
@@ -153,6 +154,29 @@ When the `@NotNull` annotation is missing from a type argument, you get a platfo
 fun toSet(elements: (Mutable)Collection<String!>) : (Mutable)Set<String!> { ... }
 ```
 
+Kotlin also takes into account nullability annotations on type arguments of base classes and interfaces. For example, 
+there are 2 Java classes with the signatures provided below.
```

Comment:
```suggestion
there are two Java classes with the signatures provided below.
```

[MS style guide](https://docs.microsoft.com/en-us/style-guide/numbers): spell out whole numbers from zero through nine

Reply:
Thank you!